src/components/Header/AlgoSearch/Search.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react"
+import { GlobalHotKeys } from "react-hotkeys"
+import { SearchButton } from "./SearchButton"
+import { SearchModal } from "./SearchModal"
+
+const hotKeysMap = {
+  toggleModal: "command+k",
```

Comment:
Does this work for windows? control+k

Reply:
This does not work on windows, I tested and confirmed

Reply:
Good catch. Added ctrl + k as a hotkey. It will still use the command icon, but I feel like it translates